src/mudlet.h

```diff
@@ -233,7 +233,7 @@ public slots:
     void slot_multi_view();
     void slot_stopAllTriggers();
     void slot_userToolBar_hovered(QAction* pA);
-    void slot_connection_dlg_finnished(const QString& profile, int historyVersion);
+    void slot_connection_dlg_finished(const QString &profile, int historyVersion);
```

Comment:
Are you sure about `&` being associated with the _parameter name_ and not the _type_ - I thought we have now set the clang-format to group `&` (and `*`) with the type specifier...? 😖